src/app/main/component/events/components/event-editor/event-editor.component.spec.ts

```
@@ -1,0 +1,96 @@
+      localStorageGetPrevPage.and.returnValue('/events');
+      fixture.detectChanges();
+
+      expect(localStorageGetPrevPage).toHaveBeenCalled();
+      expect(component.previousPath).toBe('/events');
+    });
+  });
+
+  it('sets event store service event id to one found in route', async () => {
+    const eventStoreService = TestBed.inject(EventStoreService);
+    const route = TestBed.inject(ActivatedRoute);
+    (route as any).params = of({id: 22}); // ActivatedRoute param :id = 22
```

ESLint

[error] A space is required after '{'.

(object-curly-spacing)